frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1407,14 +1406,14 @@ function ReaderFooter:addToMainMenu(menu_items)
             {
                 text_func = function()
                     local prefix_text = ""
-                    if self.settings.item_prefix == "icons" then
-                        prefix_text = _("Icons")
-                    elseif self.settings.item_prefix == "compact_items" then
-                        prefix_text = _("Compact items")
-                    elseif self.settings.item_prefix == "letters" then
-                        prefix_text = _("Letters")
+                    if self.settings.item_style == "icons" then
+                        prefix_text = _("Icon prefixes")
+                    elseif self.settings.item_style == "letters" then
+                        prefix_text = _("Letter prefixes")
+                    elseif self.settings.item_style == "compact" then
+                        prefix_text = _("Compact")
                     end
-                    return T(_("Prefix: %1"), prefix_text)
+                    return T(_("Item style: %1"), prefix_text)
```

Comment:
And this one indeed icon style.

Reply:
I had used "Item style" because Compact affects more than just the prefixes/icons, like the whitespace size and number of spaces used as a separator

Reply:
That was a typo. I meant this one's good.